src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
```

Comment:
```suggestion
import { PureComponent } from 'react';
```
Missed this line, it should be like this.